index.js

```diff
@@ -16,17 +16,27 @@ module.exports = () => {
           return;
         }
 
-        let resolvedPath = path.resolve(
-          node.params.slice(
-            `'"`.includes(node.params[0]) ? 1 : 0,
-            `'"`.includes(node.params[~-node.params.length]) ? -1 : undefined
-          )
-        );
+        let id = node.params
+          .replace(/^(url\()?['"]?/, "")
+          .replace(/['"]?(\))?$/, "");
```

Comment:
```suggestion
          .replace(/^(url\(\s*)?['"]?/, "")
          .replace(/['"]?\s*(\))?$/, "");
```

just in case there can be whitespace inside the brackets,

```css
@media (prefers-color-scheme: light) {
  :root:not([data-theme="dark"]) {
    @nested-import url(
      './test/mocks/mocks/mocks/mocks/mocks/mocks/mocks/mocks/mocks/mocks/mocks/mocks/mocks/mocks/colors1.css'
    );
    @nested-import url("./test/mocks/colors2.css");
    @nested-import url(./test/mocks/colors1.css);
  }
}
```

↑ that's still valid, right?